docs/source/setup/nicfi.rst

```diff
@@ -85,4 +85,4 @@ If you are not connected to your Google account or are connected via a different
 
 .. important::
 
-    Now you are ready to easily access and process the NICFI-Planet Basemaps in SEPAL!!  
\ No newline at end of file
+    For more information on how to use these high resolution data on SEPAL instances, please refer to the section of the `Planet academy <https://university.planet.com/path/nicfi>`__ dedicated to NICFI imagery. 
```

Comment:
For additional information that may help when processing the NICFI/Planet high resolution imagery in SEPAL, please refer to the section of the...

## This is my suggested text. If you think 'SEPAL instances' needs to remain, however...I am fine with what you have here.

Reply:
included in https://github.com/openforis/sepal-doc/pull/160/commits/f252f3a78e0487fa749937de3c3466ccb91a5e3c